exercises/forth/canonical-data.json

```diff
@@ -0,0 +1,214 @@
+{
+  "parsing and numbers": {
+    "cases": [
+      {
+        "description": "empty input results in empty stack",
+        "input": [],
+        "expected": ""
+      },
+      {
+        "description": "numbers just get pushed onto the stack",
+        "input": [
+          "1 2 3 4 5"
+        ],
+        "expected": "1 2 3 4 5"
+      },
+      {
+        "description": "all non-word characters are separators",
+        "input": [
+          "1\u00002\u00133\n4\r5 6\t7"
+        ],
+        "expected": "1 2 3 4 5 6 7"
+      }
+    ]
+  },
+  "basic arithmetic": {
+    "cases": [
+      {
+        "description": "addition and subtraction",
+        "input": [
+          "1 2 + 4 -"
+        ],
+        "expected": "-1"
+      },
+      {
+        "description": "multiplication and division",
+        "input": [
+          "2 4 * 3 /"
```

Comment:
note that this integer division. maybe that should be called out with a separate test that points this out in its description.